forge/forge.js

```diff
@@ -85,12 +85,13 @@ module.exports = async (options = {}) => {
     }
 
     if (runtimeConfig.telemetry.backend?.sentry?.dsn) {
+        const environment = process.env.SENTRY_ENV ?? (process.env.NODE_ENV ?? 'unknown')
         server.register(require('@immobiliarelabs/fastify-sentry'), {
             dsn: runtimeConfig.telemetry.backend.sentry.dsn,
-            environment: process.env.SENTRY_ENV ?? (process.env.NODE_ENV ?? 'unknown'),
+            environment,
             release: `flowfuse@${runtimeConfig.version}`,
-            tracesSampleRate: 0.1,
-            profilesSampleRate: 0.1,
+            tracesSampleRate: environment === 'production' ? 0.05 : 0.1,
+            profilesSampleRate: environment === 'production' ? 0.05 : 0.1,
```

Comment:
We may want to add custom filtering (via `tracesSampler`) to sample fewer device checkin requests, since those seem to make up a large percentage of our incoming events.